e2e/vm/soci_test.go

```diff
@@ -98,7 +98,7 @@ var testSoci = func(o *option.Option, installed bool) {
 				"- soci\nvmType: %s\nrosetta: false", vmType)))
 			command.New(o, virtualMachineRootCmd, "init").WithoutCheckingExitCode().WithTimeoutInSeconds(160).Run()
 			port = fnet.GetFreePort()
-			command.New(o, "run", "-dp", fmt.Sprintf("%d:5000", port), "--name", "registry", registryImage).
+			command.New(o, "run", "-d", "-p", fmt.Sprintf("%d:5000", port), "--name", "registry", registryImage).
```

Comment:
Why'd we have to change this?

Reply:
I changed that temporarily when I first encountered the implicit requirement to handle concatenated short flags.
I must have missed this test line when I was changing them back.

I will return this to the `-dp 8080:5000` version.